sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisModels/OperationsLiveTests.cs

```diff
@@ -148,7 +148,7 @@ public async Task BuildClassifierOperationCanPollFromNewObject()
             }
             finally
             {
-                if (sameOperation.HasValue)
+                if (sameOperation != null && sameOperation.HasValue)
```

Comment:
Updating those to get a proper error message, instead of `NullReferenceException`.